Makefile

```
@@ -0,0 +1,31 @@
+
+ARDUINO_INCLUDE=\
+								-D ARDUINO_ARCH_AVR \
+								-D ARDUINO_AVR_MEGA2560 \
+								-I deps/ArduinoCore-avr/cores/arduino/ \
+								-I deps/ArduinoCore-avr/variants/mega/ \
+
+# Find all implementation files excluding non AVR archs (eg: samd)
+EXECUTABLES= ${shell find src -type f -name *.cpp \
+						 -not -path */samd_*}
+
+.PHONY: clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)